src/main/kotlin/it/pagopa/wallet/eventdispatcher/configuration/WalletCDCClientConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package it.pagopa.wallet.eventdispatcher.configuration
+
+import it.pagopa.wallet.eventdispatcher.configuration.properties.WalletCDCConfiguration
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.common.serialization.StringSerializer
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.kafka.core.DefaultKafkaProducerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.kafka.support.serializer.ErrorHandlingDeserializer
+import org.springframework.kafka.support.serializer.JsonSerializer
+
+@Configuration
+class WalletCDCClientConfiguration {
+
+    @Bean(name = ["cdcEventHubClient"])
+    fun cdcKafkaTemplate(
+        walletCDCConfiguration: WalletCDCConfiguration
+    ): KafkaTemplate<String, Any> {
+        val configProps =
+            mapOf(
+                ProducerConfig.BOOTSTRAP_SERVERS_CONFIG to walletCDCConfiguration.bootstrapServers,
+                ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG to StringSerializer::class.java,
+                ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to
+                    ErrorHandlingDeserializer::class.java,
+                ErrorHandlingDeserializer.VALUE_DESERIALIZER_CLASS to JsonSerializer::class.java
```

Comment:
```suggestion
                ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to
                    JsonSerializer::class.java,
                ErrorHandlingDeserializer.VALUE_DESERIALIZER_CLASS to ErrorHandlingDeserializer::class.java
```
should this two be swapped or I'm missing something here?

Reply:
ops. fixed here 35d4104dcb1eb741abf4170bc809dc851ea59234